core/src/main/scala/kafka/common/Topic.scala

```diff
@@ -18,9 +18,14 @@
 package kafka.common
 
 import util.matching.Regex
-import org.apache.kafka.common.internals.TopicConstants.INTERNAL_TOPICS
+
+import scala.collection.immutable
 
 object Topic {
+
+  val GroupMetadataTopicName = "__consumer_offsets"
```

Comment:
The only other definition I see is in TestUtils and it looks like we could use an arbitrary name for an internal topic for those test cases. For example, we could use "__internal_topic".